lightsaber/processor/src/main/java/com/joom/lightsaber/processor/model/ImportPoint.kt

```diff
@@ -22,12 +22,30 @@ import com.joom.grip.mirrors.MethodMirror
 import com.joom.grip.mirrors.Type
 
 sealed class ImportPoint {
-  data class Method(val method: MethodMirror) : ImportPoint()
-  data class Field(val field: FieldMirror) : ImportPoint()
+  abstract val converter: Converter
```

Comment:
Maybe `converter` could be nullable when we can access instance directly?

Reply:
I think it will be better to have similar converter semantics as in `Injectee` class. But we can change it in both cases